### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by smile18** [mcavage/node-ldapjs#421]
I am getting compilation error for dtrace-provider and source-map-support at `ldap = require(ldapjs);`

D:\WebStormWork\myapp>npm install ldapjs --save
> dtrace-provider@0.8.0 install D:\WebStormWork\myapp\node_modules\bunyan\node_modules\dtrace-provider
> node scripts/install.js

> dtrace-provider@0.7.1 install D:\WebStormWork\myapp\node_modules\dtrace-provider
> node scripts/install.js

myapp@0.0.0 D:\WebStormWork\myapp
`-- ldapjs@1.0.1
  +-- assert-plus@1.0.0
  +-- backoff@2.5.0
  | `-- precond@0.2.3
  +-- bunyan@1.8.5
  | +-- dtrace-provider@0.8.0
  | +-- moment@2.17.1
  | +-- mv@2.1.1
  | | +-- ncp@2.0.0
  | | `-- rimraf@2.4.5
  | |   `-- glob@6.0.4
  | `-- safe-json-stringify@1.0.3
  +-- dtrace-provider@0.7.1
  +-- ldap-filter@0.2.2
  | `-- assert-plus@0.1.5
  +-- vasync@1.6.4
  | `-- verror@1.6.0
  |   `-- extsprintf@1.2.0
  `-- verror@1.9.0
    `-- extsprintf@1.3.0
	
But compilation fails at 

WARNING in ./~/dtrace-provider/dtrace-provider.js
Module not found: Error: Can't resolve './build' in 'D:\WebStormWork\myapp\node_modules\dtrace-provider'
 @ ./~/dtrace-provider/dtrace-provider.js 17:22-81
 @ ./~/ldapjs/lib/dtrace.js
 @ ./~/ldapjs/lib/server.js
 @ ./~/ldapjs/lib/index.js
 @ ./src/app/components/login/ldap.client.ts
 @ multi webpack-dev-server/client?http://localhost:4200/ ./src/main.ts

WARNING in ./~/source-map-support/source-map-support.js
Module not found: Error: Can't resolve 'module' in 'D:\WebStormWork\myapp\node_modules\source-map-support'
 @ ./~/source-map-support/source-map-support.js 474:15-32
 @ ./~/bunyan/lib/bunyan.js
 @ ./~/ldapjs/lib/index.js
 @ ./src/app/components/login/ldap.client.ts
 @ multi webpack-dev-server/client?http://localhost:4200/ ./src/main.ts
webpack: Compiled with warnings.

**Comment by donnrriz** [comment]
Yes Im getting the same error,   You should try this workaround https://github.com/trentm/node-bunyan/issues/392.

**Comment by Kunal-KP** [comment]
I am getting the same error, can anyone suggest any fix for it ?

**Comment by JD2k** [comment]
I could use webpack and this package out of the box after adding nodeExternals (https://github.com/liady/webpack-node-externals) to webpack config.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.
